packages/google_maps_flutter/ios/Classes/GoogleMapController.m

```diff
@@ -217,6 +221,12 @@ - (void)mapView:(GMSMapView*)mapView willMove:(BOOL)gesture {
 }
 
 - (void)mapView:(GMSMapView*)mapView didChangeCameraPosition:(GMSCameraPosition*)position {
+  if (!_cameraDidInitialSetup) {
+    // A temporary workaround for a bug that the camera is not properly positioned at
```

Comment:
Note that we suspect that this is an issue with the iOS Google Maps SDK(which is still being investigated), and that we call moveCamera to the current position as a temporary workaround.
Also include a TODO to remove this once the Maps SDK issue is resolved(I'm ok with closing the current Github issue with this workaround, and filing a separate one for this TODO).

Reply:
I meant note it in the code comment 😄

Reply:
updated the comments, and also opened the issue ticket.